nodeadm/internal/containerd/runtime_config_nvidia.go

```diff
@@ -0,0 +1,39 @@
+package containerd
+
+import (
+	"os"
+
+	"github.com/awslabs/amazon-eks-ami/nodeadm/internal/api"
+	"go.uber.org/zap"
+)
+
+const (
+	nvidiaRuntimeName       = "nvidia"
+	nvidiaRuntimeBinaryPath = "/usr/bin/nvidia-container-runtime"
+)
+
+func NewNvidiaRuntimeConfigMixin() *nvidiaRuntimeConfigMixin {
+	runtimeOptionMixin := nvidiaRuntimeConfigMixin{
+		runtimeBinaryName: nvidiaRuntimeBinaryPath,
+	}
+	if binPath, ok := os.LookupEnv("NODEADM_NVIDIA_CONTAINER_RUNTIME_PATH"); ok {
+		runtimeOptionMixin.runtimeBinaryName = binPath
+	}
```

Comment:
could you override `runtimeBinaryName` in your unit tests and avoid adding this fallback to prod?